packages/vscode-extension/src/utilities/minimumSuportedNodeVersion.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import { Logger } from "../Logger";
+
+/* 
+This function return a minimum supported node version for given appRoot assuming node_modules are installed,
+if node modules are not installed the function will return a default value, but it may be incorrect in future versions
+of react native, so it should be always used after node_modules installation.
+*/
+export function minimumSupportedNodeVersion(appRoot: string): string {
```

Comment:
NIT:
Functions should be verbs, not nouns.
```suggestion
export function getMinimumSupportedNodeVersion(appRoot: string): string {
```